providerframework/metaservice/backuprespository/client.go

```diff
@@ -87,12 +87,14 @@ func (repo *backupRepository) Upload(filePath string) error {
 		Data: &pb.UploadRequest_Info{
 			Info: &pb.UploadRequest_FileInfo{
 				FileIdentifier: path.Base(filePath),
+				Attributes:     attributes,
 			},
 		},
 	})
 	if err != nil {
 		return err
 	}
+	log.Infof("88888attributes are:8888 %v", attributes)
```

Comment:
please do change the logs properly.....don't add 888 etc . please do change this all the places

Reply:
changed